src/main/kotlin/com/cognifide/gradle/common/http/HttpClient.kt

```diff
@@ -140,6 +147,14 @@ open class HttpClient(private val common: CommonExtension) {
             )
         }
 
+        if (!bearerToken.orNull.isNullOrBlank()) {
+            val headers = listOf(
+                BasicHeader(HttpHeaders.CONTENT_TYPE, "application/json"),
```

Comment:
this is wrong. you cannot add this header here as common plugin is general; not your case-specific

Reply:
removed